pkg/api/v1/router.go

```diff
@@ -121,6 +121,24 @@ func (r *Router) Routes(rg *gin.RouterGroup) {
 		srvCmpntFwSets.DELETE("/:uuid", amw.RequiredScopes(deleteScopes("server-component-firmware-sets")), r.serverComponentFirmwareSetDelete)
 		srvCmpntFwSets.POST("/:uuid/remove-firmware", amw.RequiredScopes(deleteScopes("server-component-firmware-sets")), r.serverComponentFirmwareSetRemoveFirmware)
 	}
+
+	// /bom-info
+	srvBoms := rg.Group("/bom-info")
+	{
+		// /bom-info/batch-boms-upload
+		uploadXlsxFile := srvBoms.Group("/batch-boms-upload")
```

Comment:
can we call this endpoint `/batch-upload` ? I suggest removing the `XlsxFile` from the variable name, since this will now accept a JSON payload